.github/workflows/triage.yml

```diff
@@ -15,7 +15,7 @@ jobs:
             const { issue, repository } = context.payload;
             const { number, body } = issue;
             const { owner, name } = repository;
-            const regex = /###\sRelated\scomponent\n\n(\w*)\n/gm;
+            const regex = /###\sRelated\scomponent\n\n(\w.*)\n/gm;
```

Comment:
`.` seems too wide of match to just support spaces. Perhaps `([\w\s]*)` is more suitable?

Reply:
I don't think we need to be this rigid, it would match any label from our list and apply it only if it exists.